File: app/src/main/java/com/saitejajanjirala/imdbclone/models/NowPlaying.java
package com.saitejajanjirala.imdbclone.models;

import com.saitejajanjirala.imdbclone.BaseApplication;

import java.util.List;

public class NowPlaying extends BaseResponse{

    public Dates dates;
    public int page;
    public List<Result> results;
    public int total_pages;
    public int total_results;

    class Dates{
        public String maximum;
        public String minimum;

        @Override
        public String toString() {
            return "Dates{" +
                    "maximum='" + maximum + '\'' +
                    ", minimum='" + minimum + '\'' +
                    '}';
        }
    }

    @Override
    public String toString() {
        return "NowPlaying{" +
                "dates=" + dates +
                ", page=" + page +
                ", results=" + results +
                ", total_pages=" + total_pages +
                ", total_results=" + total_results +
                '}';
    }
}

File: app/src/main/java/com/saitejajanjirala/imdbclonekotlin/models/NowPlaying.kt
package com.saitejajanjirala.imdbclonekotlin.models


data class NowPlaying(
    var dates: Dates? = null,
    var page: Int = 0,
    var results: List<Result>? = null,
    var total_pages: Int = 0,
    var total_results: Int = 0,
):BaseResponse() {
    inner class Dates {
        var maximum: String? = null
        var minimum: String? = null
        override fun toString(): String {
            return "Dates{" +
                    "maximum='" + maximum + '\'' +
                    ", minimum='" + minimum + '\'' +
                    '}'
        }
    }
}

